docs/ARCHITECTURE.md

```diff
@@ -1,44 +1,43 @@
-# Architecture
```

Comment:
Not sure if this page qualifies as "ARCHITECTURE"

Reply:
not sure me too. but my point was to move it to more approciate place under more upprociate owners, not to name it right (so somebody can rename it right later).

Reply:
will resolve. feel free to reopen if any questions. we may fix name in next PR.